src/pydap/handlers/dap.py

```diff
@@ -439,8 +456,8 @@ def __repr__(self):
     def __getitem__(self, index):
         # build download url
         index = combine_slices(self.slice, fix_slice(index, self.shape))
-        scheme, netloc, path, params, query, fragment = urlparse(self.baseurl)
-        ce = "dap4.ce=" + quote(self.id) + hyperslab(index) + query
+        scheme, netloc, path, _, query, fragment = urlparse(self.baseurl)
+        ce = "dap4.ce=" + self.id + hyperslab(index)
```

Comment:
`query` does not need to be passed  - variable names are already incorporated within `self.id`

Reply:
also the `_` implies the third element of a tuple is released from memory (not assigned to any variable)